eos/utils/kmatrix_TEST.cc

```diff
@@ -26,63 +27,64 @@
 using namespace test;
 using namespace eos;
 
-///////////////////////
+/////////////////////////////////////////////
 /// Simplest K matrix, 1 channel, 1 resonance
-///////////////////////
+/////////////////////////////////////////////
 struct PPchan11 :
     public KMatrix<1,1>::Channel
 {
-    PPchan11(std::string name, double m1, double m2, unsigned N_orbital, const Parameters & p) :
-        Channel(name, m1, m2, N_orbital, {{ p["test::g0_1"] }})
+    PPchan11(std::string name, Parameter m1, Parameter m2, unsigned l_orbital, const Parameters & p) :
+        Channel(name, m1, m2, l_orbital, p["test::q0"], {{ p["test::g0_1"] }})
     {
     };
 
-    // Useful definitions for beta and rho
     double mm = this->_m1 - this->_m2;
     double mp = this->_m1 + this->_m2;
     // sqrt of the Källen factor, defined with an absolute value
-    double sqlk(const double & s)
+    double sqlk(const complex<double> & s)
     {
         return std::sqrt(std::abs((s - mp * mp) * (s - mm * mm)));
     }
     const double pi = M_PI;
     const complex<double> i = complex<double>(0.0, 1.0);
 
-    double beta(const double & s)
+    complex<double> rho(const complex<double> & s)
     {
-        if (s < mp * mp)
+        if (real(s) < mp * mp)
             return 0.;
         else
-            return sqlk(s) / s;
+            return sqlk(s) / abs(s) / 16.0 / pi;
     }
 
-    complex<double> rho(const double & s)
+    complex<double> chew_mandelstam(const complex<double> & S)
     {
+        double s = real(S);
```

Comment:
Wait, what? Are you computing the CMs only for the real part of ``s``?

Reply:
Only for this example, in the ee->ccbar channels, the CM are correctly implemented for complex s